tests/test_units.py

```diff
@@ -92,7 +92,10 @@ def test_convert_unit_with_custom_registry(test_df):
 
     # Using "-equiv" after a unit should make no difference
     ('AR4GWP100', 'co2-equiv', 1.),
-    ('AR5GWP100', 'ch4-equiv', 28)
+    ('AR5GWP100', 'ch4-equiv', 28),
+
+    # Converting C -> CO2e should work
+    ('AR4GWP100', 'C', 11 / 3)
```

Comment:
conversion from C to CO2 or CO2-eq shouldn't require a context at all, see https://github.com/IAMconsortium/units/issues/23 - adding a todo as a reminder to update the test

```suggestion
    # TODO remove context once IAMconsortium/units#23 is resolved
    ('AR4GWP100', 'C', 11 / 3)
```